google-cloud-kubernetes/config.json

```diff
@@ -0,0 +1 @@
+{"config-changes":{"Xmx":"20G","retrieval.forbackend.concurrency":"5","retrieval.forbackend.loops":"30"},"num-nodes":"1","node-type":"n1-standard-8"}
```

Comment:
What if user wants different configurations? Change the configurations in the file?